HL347M_L123PZ/Program.cs

```diff
@@ -0,0 +1,7 @@
+﻿using StudentTerminal.Commands;
+
+#if NEW // Feltölti a JSON file-t új randomizált adatokkal
```

Comment:
őőő :D Ezt pl meg lehetett volna oldani konzol argumentumból is, mert így ez fordítási szimbólum.